lib/applications/resolvers.ts

```diff
@@ -717,6 +743,47 @@ export const updateApplicationGeneralInformation: Resolver<
   return { ok: true };
 };
 
+/**
+ * Update the general information section of an application
```

Comment:
```suggestion
 * Update the general information section of a new application (includes date of birth, gender)
```
Let's make the distinction with `updateApplication` more clear. I'm also not sure if making a new resolver specifically for new applications is necessary/a good idea. It will make it easier to validate schemas at the cost of redundancy. It seems like we would deal with some TS compilation and GraphQL validation errors otherwise, so let's keep it for now. We can revisit this when we work on form validation

Reply:
Yeah I agree that its not the cleanest solution, but yeah without it we get lots of issues with type validation. Good to revisit this then

Reply:
Sounds good, created a ticket in Notion